test/school_test.rb

```diff
@@ -45,11 +45,37 @@ def test_it_can_add_student_names
   end
 
   def test_it_can_calculate_end_time
-    skip
+    # skip
     school1 = School.new('9:00', 7)
     school2 = School.new('9:00', 3)
 
     assert_equal '16:00', school1.end_time
     assert_equal '12:00', school2.end_time
   end
+
+  def test_school_can_be_full_time
+    school1 = School.new('9:00', 7)
+    school2 = School.new('9:00', 3)
+
+    assert school1.is_full_time?
+    refute school2.is_full_time?
+  end
+
+  def test_school_has_standard_student_names
+    school = School.new('9:00', 7)
+
+    school.add_student_name('Aurora')
+    school.add_student_name('tim')
+    school.add_student_name('megan')
+
+    assert_equal ["Aurora", "Tim", "Megan"], school.standard_student_names
+  end
+
+  def test_school_converts_end_time_to_clock_time
+    school1 = School.new('9:00', 7)
+    school2 = School.new('9:00', 3)
+
+    assert_equal "4:00", school1.convert_end_time_to_clock_time
```

Comment:
As a general rule you probably only should have one assertion per test.  Line 78 gives me enough confidence that this method works